gradle/dependencyManagement.gradle

```diff
@@ -0,0 +1,155 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencyManagement {
+  /** Definition of dependencies versions */      
+  def hadoop_version = getVersion("hadoop_version", "1.0.4")
+  def protobuf_version = getVersion("protobuf_version", "2.4.1")
+  def jets3t_version = getVersion("jets3t_version", "0.7.1")
+  
+  if (project.hasProperty('hadoop-0.23')) {
+    hadoop_version = "0.23.10"
+  }
+  if (project.hasProperty('hadoop-2.2')) {
+    hadoop_version = "2.2.0"
+    protobuf_version = "2.5.0"
+    jets3t_version = "0.7.1"
+  }
+  if (project.hasProperty('hadoop-2.3')) {
+    hadoop_version = "2.3.0"
+    protobuf_version = "2.5.0"
+    jets3t_version = "0.9.0"
+  }
+  if (project.hasProperty('hadoop-2.4')) {
+    hadoop_version = "2.4.0"
+    protobuf_version = "2.5.0"
+    jets3t_version = "0.9.0"
+  }
+
+  def spark_version = getVersion("spark_version", "1.1.1")
```

Comment:
Here the idea is to centralise the dependencies so it is more easy to control  the version.
I detached it from the build because its too massive and since we dont change version often for me this is a perfect place to be.